numpy/lib/_function_base_impl.py

```diff
@@ -2335,15 +2335,15 @@ class vectorize:
     ...     while _p:
     ...         res = res*x + _p.pop(0)
     ...     return res
-    >>> vpolyval = np.vectorize(mypolyval, excluded=['p'])
-    >>> vpolyval(p=[1, 2, 3], x=[0, 1])
-    array([3, 6])
 
-    Positional arguments may also be excluded by specifying their position:
+    Here, we exclude the zeroth argument from vectorization whether it is
+    passed by position or keyword.
 
-    >>> vpolyval.excluded.add(0)
```

Comment:
I think this is an improvement because it shows the use of `excluded` with the documented type (set) and excludes the zeroth argument whether passed by keyword or position from the outset (rather than treating positional use as an afterthough). This eliminated the example that demonstrates modification of the `excluded` attribute of the wrapped function... but it's unclear whether that was intended to be a public feature in the first place, since it is not documented otherwise.